### ISSUE TITLE
Cache clearers

### ISSUE THREAD

**Issue by sagikazarmark** [php-cache/cache-bundle#18]
Hey guys,

I am not really familiar with symfony caching, but wouldn't it be a good idea to provide cache warmers as well?

**Comment by aequasi** [comment]
Cache warmers are something that the user really has to implement. There's not really a way for us to cache everything for you. Unless you have an idea.

**Comment by sagikazarmark** [comment]
Sorry, I actually meant cache clearers.

**Comment by aequasi** [comment]
Thats already implemented with the [CacheFlushCommand](https://github.com/php-cache/cache-bundle/blob/master/src/Command/CacheFlushCommand.php)

**Comment by sagikazarmark** [comment]
Yeah, but I meant the core symfony cache cleaning command.

**Comment by aequasi** [comment]
I don't know what you are trying to suggest

**Comment by aequasi** [comment]
@sagikazarmark can you clarify?

**Comment by sagikazarmark** [comment]
Would be nice to automatically clear the cache for the default symfony cache clear command.

`$ php bin/console cache:clear`

**Comment by aequasi** [comment]
Problem is, some people may not want to clear both at the same time, and as far as I know, we can't add a flag to the symfony command to do that. Thoughts @Nyholm?

**Comment by Nyholm** [comment]
Hm. It is dangerous to add functionality to an existing command. Like @aequasi says, some might not want to clear both caches. We could allow for clearing the symfony cache via `cache:flush`. 

```bash
$ php app/console cache:flush help  
Usage: php app/console cache:flush [options]
Available options are: 
 --all               Clears all the caches
 --symfony           Clears symfony cache. Same as cache:clear
 --doctrine          Clear doctrine cache
 --session           Clear session cache
 --router            Clear router cache
 --provider name     Clears cache for a specific provider
```

**Comment by sagikazarmark** [comment]
Like the idea. The point would be you have the possibility to clear out everything.

**Comment by Nyholm** [comment]
If #30 gets merged you may clear all caches with: 
``` bash
$ php bin/console cache:flush all
```
Or..
```bash 
$ php bin/console cache:flush
Do you really want to clear all caches ?
> y
```

**Comment by sagikazarmark** [comment]
:+1:

**Comment by Nyholm** [comment]
Closed by #30
